go.mod

```diff
@@ -8,18 +8,18 @@ require (
 	github.com/drone/envsubst v1.0.3
 	github.com/go-logr/logr v1.4.2
 	github.com/google/go-cmp v0.6.0
-	github.com/jaypipes/ghw v0.12.0
-	github.com/k8stopologyawareschedwg/deployer v0.21.3
+	github.com/jaypipes/ghw v0.16.0
+	github.com/k8stopologyawareschedwg/deployer v0.22.0
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.1.2
 	github.com/k8stopologyawareschedwg/podfingerprint v0.2.2
-	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.20.3
+	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.21.0
 	github.com/kubevirt/device-plugin-manager v1.19.4
 	github.com/mdomke/git-semver v1.0.0
-	github.com/onsi/ginkgo/v2 v2.21.0
-	github.com/onsi/gomega v1.35.1
-	github.com/openshift/api v0.0.0-20240524162738-d899f8877d22
-	github.com/openshift/hypershift/api v0.0.0-20241115183703-d41904871380
-	github.com/openshift/machine-config-operator v0.0.1-0.20230724174830-7b54f1dcce4e
+	github.com/onsi/ginkgo/v2 v2.22.0
+	github.com/onsi/gomega v1.36.1
+	github.com/openshift/api v0.0.0-20250305013520-e7f23be12279 // release-4.18
+	github.com/openshift/hypershift/api v0.0.0-20241115183703-d41904871380 // release-4.18
+	github.com/openshift/machine-config-operator v0.0.1-0.20230724174830-7b54f1dcce4e // release-4.13
```

Comment:
too old, we need to update also this one: PTAL to https://github.com/openshift-kni/numaresources-operator/pull/1259/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R154

Reply:
I think we need https://github.com/openshift-kni/numaresources-operator/pull/1258 before becuase if updated the helpers won't be found in the new version